docs/Simulators.md

```diff
@@ -0,0 +1,85 @@
+# Simulators
+
+Inductiva API has available several open-source simulators ready to use. Users who are familiar with the simulators can easily start running simulations with their previously prepared simulation configuration files. In this way, they can take advantage of performant hardware to speed up their simulation and exploration.
+No installation or management of the simulators is required, no need to worry about hardware or software dependencies. Inductiva API takes care of all of that for you.
+
+The simulators currently available are all open-source and have their dedicated documentation:
+- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
+- [DualSPHysics](https://dual.sphysics.org/)
+- [OpenFOAM](https://www.openfoam.com/)
+- [SWASH](https://swash.sourceforge.io/)
+- [XBeach](https://oss.deltares.nl/web/xbeach/)
+- [Reef3D](https://reef3d.wordpress.com/)
+- [GROMACS](https://www.gromacs.org/)
+- [FDS](https://pages.nist.gov/fds-smv/)
+
+Check the documentation of each simulator to learn more on how to configure them. Here, we highlight how to use your preferred simulator via Inductiva API and how to scale your simulations with simplicity. There is a general structure that all simulators follow, and then there are some specificities for each simulator.
+
+## Simulators via Inductiva API
+
+To run a simulation, prepare the configuration files for your desired simulator and place them in a designated folder. The simulator will use this folder to perform the simulation.
+
+### Example
+
+For a first simulation, we run a coastal dynamics simulation with SWASH.
+
+```python
+import inductiva
+
+input_dir = inductiva.utils.files.download_from_url(
```

Comment:
A couple of things here:

1)  inductiva.utils.files.download_from_url is too deep. It should be just  inductiva.utils.download_from_url or  inductiva.files.download_from_url

2) there is a magic unzip happening here.